lang/en/docs/materials-designer/header-menu/advanced/boundary-conditions.md

```diff
@@ -0,0 +1,30 @@
+# Boundary Conditions
+
+Boundary conditions specify how the system under investigation (referred to as "Slab" for the case of interfaces) can interact or is related to its surroundings. Such customization of boundary conditions can be especially resourceful in the case of [Effective Screening Medium](../../../models/auxiliary-concepts/esm.md) calculations.
+
+## Set Boundary Conditions Dialog
+
+Open the "Set Boundary Conditions" dialog via the ["Advanced" menu](../advanced.md) within the Materials Designer interface. The dialog has the appearance as highlighted in the image below.
+
+![Boundary Conditions Dialog](../../../images/materials-designer/boundary-conditions.png "Boundary Conditions Dialog")
```

Comment:
You put a good explanation of QE input parameters, let’s move the offset to Boundary Conditions page and refer back to it from the tutorial.

We also need to add a picture of a material with the ESM boundary conditions visualized and explain the coloring of the actual boundary conditions - eg. yellow = vacuum, blue = metal.